_includes/header.html

```diff
@@ -88,7 +88,27 @@
 										<a href="https://gci14.fossasia.org">FOSSASIA GCI 2014/15</a>
 									</li>
 								</ul>
-							</li>	
+							</li>
+              <li>
```

Comment:
Can you please intend the code? It will be great. Rest looks fine.

Reply:
Sure. Will make changes :blush: